client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -475,7 +475,7 @@ class WpcomChecklistComponent extends PureComponent {
 			<TaskComponent
 				{ ...baseProps }
 				bannerImageSrc="/calypso/images/stats/tasks/create-tagline.svg"
-				completedButtonText={ translate( 'Change' ) }
+				completedButtonText={ translate( 'Edit tagline' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Editing Tagline', { context: 'Customizer editing element status message'} )` *ES Score: 9*